config/config.exs

```diff
@@ -35,6 +35,21 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+config :esbuild,
+  version: "0.23.1",
```

Comment:
I'm assuming this is the latest version?

Reply:
yep